Formula/softhsmv2.rb

```diff
@@ -0,0 +1,31 @@
+class Softhsmv2 < Formula
+  desc "SoftHSM version 2"
```

Comment:
Descriptions should not start with the formula name, which this one effectively, if not literally, does.